src/main/java/com/minecolonies/coremod/colony/buildings/BuildingStonemason.java

```diff
@@ -55,7 +56,7 @@ public View(final ColonyView c, final BlockPos l)
         }
 
         @NotNull
-        public com.blockout.views.Window getWindow()
+        public Window getWindow()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1161)